test/e2e/enveloped_object_placement_test.go

```diff
@@ -481,11 +481,8 @@ var _ = Describe("Process objects with generate name", Ordered, func() {
 	})
 
 	AfterAll(func() {
-		// Remove the CRP.
-		cleanupCRP(crpName)
-
-		// Clean the placed resources.
-		cleanWorkResourcesOnCluster(allMemberClusters[0])
+		By(fmt.Sprintf("deleting placement %s and related resources", crpName))
+		ensureCRPAndRelatedResourcesDeleted(crpName, allMemberClusters)
```

Comment:
why it's related to this AfterAll func?

This CRP only selects one cluster. What's the diff?

Reply:
The namespace and configmap are created on the hub cluster. The original cleanup only deletes the CRP and ensure member cluster is cleaned up, not the hub. That's why the next test scheduled on the same thread trying to create a namespace reporting namespace already exists error.

Reply:
I get that the original test was not right but how did it pass if we increase the wait time as in your another PR?

Reply:
This test was introduced 5 days ago: https://github.com/Azure/fleet/pull/1086. My another PR was created and tested before that.